### PR TITLE
apps/examples/testcase/le_tc/filesystem: Add MTD path depending on specific board

### PR DESCRIPTION
**Pull request by KimDongEon**
Add MTD path depend on imxrt1020

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [00b38dafc5d549cb8a4aaf834880bc6f4a99485f] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [00b38dafc5d549cb8a4aaf834880bc6f4a99485f] - Code Rule Check OK.

File: apps/examples/testcase/le_tc/filesystem/fs_main.c [hunk 1]
```diff
@@ -134,7 +134,13 @@
 #define INV_FD -3
 
 #define MTD_CONFIG_PATH "/dev/config"
+#if defined(CONFIG_ARCH_BOARD_CY4390x)
+#define MTD_FTL_PATH "/dev/mtdblock4"
+#elif defined(CONFIG_ARCH_BOARD_IMXRT1020_EVK)
+#define MTD_FTL_PATH "/dev/smart0p6"
```
Comment (sunghan-chang) [inline apps/examples/testcase/le_tc/filesystem/fs_main.c]: why 6?

Comment (KimDongEon) [reply apps/examples/testcase/le_tc/filesystem/fs_main.c]: Userfs is registered as 6 during the partitioning process.

Comment (sunghan-chang) [reply apps/examples/testcase/le_tc/filesystem/fs_main.c]: In specific defconfig, it is. But it does not guarantee all of defconfig has smartfs with 6 index. To make this working, you should enable automount with mountpoint.

Comment (KimDongEon) [reply apps/examples/testcase/le_tc/filesystem/fs_main.c]: In this code, the MTD_FTL_PATH define is used as a test path for reading and writing directly to the MTD block dev.

Therefore, to run tc, you need the dev path before mounting.

However, registration numbers are registered in the order defined in CONFIG_FLASH_PART_TYPE in the defconfig file. Is there a way to unify this order with other defconfigs?

**Pull request merged**